src/ResourceManager/Network/Commands.Network/AzureRM.Network.psd1

```diff
@@ -338,7 +338,20 @@ CmdletsToExport = 'Add-AzureRmApplicationGatewayAuthenticationCertificate',
                'Get-AzureRmServiceEndpointPolicyDefinition', 
                'Set-AzureRmServiceEndpointPolicyDefinition', 
                'Add-AzureRmServiceEndpointPolicyDefinition', 
-               'Set-AzureRmServiceEndpointPolicy'
+               'Set-AzureRmServiceEndpointPolicy',
```

Comment:
you ned to add the <Verb>-Az<Noun> version of these cmdlets to https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Network/Commands.Network/Az.Network.psd1